### PR TITLE
My ninth homework commit.

### PR DESCRIPTION
**Pull request by mariianakurta**
My ninth homework was added.

### CODE REVIEW

**Comment by mariianakurta** [comment]
My 9th and 10th homeworks were added.

File: ALevelHomework9/ALevelHomework9/Program.cs
Comment (petrochernykh) [inline ALevelHomework9/ALevelHomework9/Program.cs:1]: You should try to use IoC

File: ALevelHomework9/ALevelHomework9/Logger.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        public void LoggerMaintenance()
+        {
+            Console.WriteLine();
+        }
+
+        public void Log(string logType, string message)
+        {
+            string logTime = DateTime.Now.ToString("MM/dd/yyyy hh:mm:ss.fff tt");
+            Console.WriteLine($"{logTime}: {logType}: {message}");
+        }
```
Comment (petrochernykh) [inline ALevelHomework9/ALevelHomework9/Logger.cs:35]: We can make one method that will be responsible for log, in this case we need to have LogLevel enum as an input parameter

File: ALevelHomework10/ALevelHomework10/ElectricalAppliancesClass.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ALevelHomework10
+{
+    internal static class ElectricalAppliancesClass
+    {
+        public static double CalculatePowerConsumption(this ElectricalDevice[] appliances)
```
Comment (petrochernykh) [inline ALevelHomework10/ALevelHomework10/ElectricalAppliancesClass.cs:11]: What if our device is switched off or plugged out

File: ALevelHomework10/ALevelHomework10/Program.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        if (appliancesLimit != null)
+        {
+            Console.WriteLine($"Appliances matching the power range of {minLimit} to {maxLimit} electric charge were found: {appliancesLimit.Name}");
+        }
+        else
+        {
+            Console.WriteLine($"No appliances matching the power range of {minLimit} to {maxLimit} electric charge were found");
+        }
+
+        allDevices.DisplayWhiteDevices();
```
Comment (petrochernykh) [inline ALevelHomework10/ALevelHomework10/Program.cs:46]: It's better to move logic to a special service

File: ALevelHomework10/ALevelHomework10/Program.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    private static void Main(string[] args)
+    {
+        Computer computer = new Computer("Computer", 2000, true, "Black");
+        Kettle kettle = new Kettle("Kettle", 500, false, "White");
+        Microwave microwave = new Microwave("Microwave", 1200, true, "White");
+
+        ElectricalDevice[] allDevices = { computer, kettle, microwave };
+
+        string largeSize = "Large";
+        var devicesSize = allDevices.Where(appliance => appliance.GetSize() == largeSize).ToArray();
```
Comment (petrochernykh) [inline ALevelHomework10/ALevelHomework10/Program.cs:19]: What will be in this variable, I am not sure that devices size will be here

File: ALevelHomework10/ALevelHomework10.sln
Comment (petrochernykh) [inline ALevelHomework10/ALevelHomework10.sln:1]: You should use some system to group al these files

**Pull request merged**